libraries/AP_Notify/AP_Notify.h

```diff
@@ -111,10 +122,17 @@ class AP_Notify
     static const struct AP_Param::GroupInfo var_info[];
 
     bool buzzer_enabled() const { return _buzzer_enable; }
+
+    static void send_text(const char *str) { snprintf(_send_text, sizeof(_send_text), "%s", str); }
```

Comment:
The send_text() is a static method so it unable access _display_type parameter variable which is instance one. Unsure I can make _display_type as a static. Also send_text() just prints string to the buffer. It is not a problem, because in case of _display_type == DISPLAY_OFF the buffer will not used to send data to display.